### DISCUSSION TITLE
ESP8266 - Attiny85 RIG

### DISCUSSION THREAD

**Discussion by Gersh5000** [JK-Rolling/DuinoI2C_ESP#7] [Show and tell]
The PCB design is tailored for Duino Coin mining using the ESP8266 master and up to 20 Attiny85 workers. I2C from master to workers runs over LLC.

![Screenshot_4](https://github.com/user-attachments/assets/aee30d4b-14ff-4225-b5ef-564f29c11057)
![IMG_1624](https://github.com/user-attachments/assets/dc8d473a-90e0-404c-99cc-e7c09a6820fb)
![IMG_1627](https://github.com/user-attachments/assets/0e3cfc73-264b-425f-b0a6-db6fdb1e77aa)

If you like it you can check it out [here](https://github.com/Gersh5000/DuinoI2C-PCB) .

**Comment by JK-Rolling** [comment]
awesome mining rig. so many 8 legged attiny spiders! 🕷️
